<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportEventCasesVerificationSetup</class>
 <widget class="QDialog" name="ReportEventCasesVerificationSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="chkSendIEMK">
     <property name="text">
      <string>Передано в ИЭМК</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания периода</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbAidType">
     <item>
      <property name="text">
       <string>не задано</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>амбулаторный</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>стационарный</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CDateEdit" name="edtBegDate"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала периода</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtEndDate"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblAidType">
     <property name="text">
      <string>Тип медицинской помощи</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="chkHideHeader">
     <property name="text">
      <string>Скрывать шапку</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="chkCompareUsishCode">
     <property name="text">
      <string>Учитывать код ЕГИСЗ Типа События</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="chkOnlyOMC">
     <property name="text">
      <string>Только ОМС</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library.DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbAidType</tabstop>
  <tabstop>chkSendIEMK</tabstop>
  <tabstop>chkHideHeader</tabstop>
  <tabstop>chkCompareUsishCode</tabstop>
  <tabstop>chkOnlyOMC</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportEventCasesVerificationSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportEventCasesVerificationSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
